### PR TITLE
Format some sections

### PR DESCRIPTION
**Pull request by scripthunter7**
Do not merge this pull request.

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -907,7 +907,7 @@ boards.4channel.org,boards.4chan.org##.adc-resp-bg
 ||fmmods.com/bnbanner/
 fmmods.com##.rbc-container > center > h5
 xero.porn###popad
-xero.porn##div[onclick^="$\(\"#videoAd"]
+xero.porn##div[onclick^="$(\"#videoAd"]
```
Comment (zloyden) [inline BaseFilter/sections/specific.txt:910]: This change is strange, but looks like that this rule is obsolete. Could you remove it?

Comment (scripthunter7) [reply BaseFilter/sections/specific.txt]: Removed

Comment (Alex-302) [reply BaseFilter/sections/specific.txt]: the rule is not obsolete, and modified rule is still working when `"` is only escaped.

Comment (Alex-302) [reply BaseFilter/sections/specific.txt]: Also no need to check for obsolete rules in this PR.

**Review by zloyden — APPROVED** [review]

File: BaseFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply BaseFilter/sections/specific.txt]: I got it, but where the rule is needed?
Anyway, looks ok.

Comment (Alex-302) [reply BaseFilter/sections/specific.txt]: On the page with player.

File: BaseFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1430,32 +1430,32 @@ mumbaimirror.indiatimes.com#?#.latestCmmtData > .mrtoday > div[class*=" "]:has(>
...
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39779
-mangarock.com#?##page-content > .container > .row > .col-12 > span[style]:has(>a[href="/membership"])
+mangarock.com#?##page-content > .container > .row > .col-12 > span[style]:has(> a[href="/membership"])
 mangarock.com#?#div[role="presentation"] > div[id^="vertical-page-"]:has(> div[class]:not([style]) > a[href="/membership"])
 mangarock.com#?#div[role="presentation"] > div[id^="vertical-page-"]:last-child > div[class] > div[class]:has(> div > a[href="/membership"])
 mangarock.com#$?#div.slick-track > div.slick-slide:has(> a[href="/membership"]) { remove: true; }
 mangarock.com#?#div.slick-track > div.slick-slide:last-child > div[class]:has(> div > a[href="/membership"])
 ! https://github.com/AdguardTeam/AdguardFilters/issues/42696
-camera.aeonax.com#$?##wrapper > #predownloads:has(> .inner > #AdamWr) { height: 0px!important; }
+camera.aeonax.com#$?##wrapper > #predownloads:has(> .inner > #AdamWr) { height: 0px !important; }
```
Comment (Sergey-Lyapin) [inline BaseFilter/sections/css_extended.txt:1458]: Do we need a space here? Almost all of such rules do not have it.

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request closed without merging**